### DISCUSSION TITLE
Why studio row size cannot make 5 row?

### DISCUSSION THREAD

**Discussion by myagizmaktav** [supabase/supabase#21398] [Feature Requests]
Hello, I want to make only see 5 row on studio because I put sitemaps array list.
For this reason some data cells about 2 mb on my table.
When I opening sitemapTable. its stuck not opening.
Can we improve this pls?

![image](https://github.com/supabase/supabase/assets/115627863/0eccaf8d-7baf-4510-8b18-4a8401e7ee09)
![image](https://github.com/supabase/supabase/assets/115627863/73b59f85-0183-4972-9908-268649fd492b)

**Comment by myagizmaktav** [comment]
And some times I problem with 1 body exceed limit how I solve this ?

**Comment by Hallidayo** [comment]
Hi @myagizmaktav - Have you got a video of this?

**Reply by myagizmaktav** [reply]
sorry I havent but data like that 
`[
'google.com/1',
'google.com/2',
'google.com/3',
'google.com/4',
'google.com/6',
...296.318
]
